### PR TITLE
zh:sync /setup/additional-setup/sidecar-injection/

### PR DESCRIPTION
**Pull request by windsonsea**
Sync with the en version:
```
content/zh/docs/setup/additional-setup/sidecar-injection/index.md
```
- [x] Docs

### CODE REVIEW

File: content/zh/docs/setup/additional-setup/sidecar-injection/index.md [hunk 1]
```diff
@@ -203,17 +212,47 @@ spec:
...
+      limits:
+        memory: 5Gi
+      requests:
+        cpu: 200m
+        memory: 5Gi
+      securityContext:
+        allowPrivilegeEscalation: false
+{{< /text >}}
+
+### 自定义模板（试验特性）{#custom-templates-experimental}
```
Comment (zirain) [inline content/zh/docs/setup/additional-setup/sidecar-injection/index.md:249]:
```suggestion
### 自定义模板（试验特性）{#customizing-injection}
```

keep anchor with En doc

File: content/zh/docs/setup/additional-setup/sidecar-injection/index.md [hunk 2]
```diff
@@ -203,17 +212,47 @@ spec:
...
 
 {{< warning >}}
 此功能为试验特性功能，可随时更改或删除。
 {{< /warning >}}
 
-可以在安装时定义一个自定义模板，例如，将 `GREETING` 环境变量注入到 `istio-proxy` 容器中：
+可以在安装时定义一个完全自定义的模板，例如，将 `GREETING` 环境变量注入到 `istio-proxy` 容器中：
```
Comment (zirain) [inline content/zh/docs/setup/additional-setup/sidecar-injection/index.md:255]: the following part is not synced to the latest doc, a mistake?

**Review by zirain — CHANGES_REQUESTED** [review]

File: content/zh/docs/setup/additional-setup/sidecar-injection/index.md (hunk 1 above)
Comment (windsonsea) [reply content/zh/docs/setup/additional-setup/sidecar-injection/index.md:249]: Feel a little strange. The original heading is 

### Custom templates (experimental)

Comment (zirain) [reply content/zh/docs/setup/additional-setup/sidecar-injection/index.md:249]: sorry, it's my mistake, ignore this.

File: content/zh/docs/setup/additional-setup/sidecar-injection/index.md [hunk 3]
```diff
@@ -233,4 +273,6 @@ spec:
                 value: hello-world
 {{< /text >}}
 
-默认情况下，Pod 会使用 `sidecar` 注入模板，自动创建模板。这可通过 `inject.istio.io/templates` 注释来替换。例如要应用默认模板和自定义模板，您可以设置 `inject.istio.io/templates=sidecar,custom`。
+默认情况下，Pod 会使用 `sidecar` 注入模板，自动创建模板。这可通过 `inject.istio.io/templates` 注解来替换。例如要应用默认模板和自定义模板，您可以设置 `inject.istio.io/templates=sidecar,custom`。
```
Comment (zirain) [inline content/zh/docs/setup/additional-setup/sidecar-injection/index.md:276]:
```suggestion
默认情况下，`istio` 会使用自动创建的  `sidecar`  模板对 Pod 进行注入。这可通过 `inject.istio.io/templates` 注解来替换。例如要应用默认模板和自定义模板，您可以设置 `inject.istio.io/templates=sidecar,custom`。
```

**Review by zirain — APPROVED** [review]
lgtm, thanks

**Pull request merged**